### PR TITLE
Update to 2021a. JB#48236

### PR DESCRIPTION
**Pull request by pvuorela**
See individual commits. Biggest shortcoming here is that wikipedia used as source for MCC information had the page split into multiple ones in 2019. But maybe better first do update to this point where it's easier and handle the rest separately.

@Tomin1 @spiiroin

### CODE REVIEW

File: scripts/zone-generate.sh [hunk 1]
```diff
@@ -84,14 +77,14 @@ rm -rf $output $signatures $md5sum && mkdir -p $output
...
 
 for i in $input ; do
   echo "Processing '$i'"
-  $zic -d $output -L /dev/null -y $yearistype $i 2> $build_dir/stderr
-  $zic -d $output/posix -L /dev/null -y $yearistype $i 2>> $build_dir/stderr
-  $zic -d $output/right -L $leapseconds -y $yearistype $i 2>> $build_dir/stderr
+  $zic -d $output -L /dev/null $i #2> $build_dir/stderr
+  $zic -d $output/posix -L /dev/null $i # 2>> $build_dir/stderr
+  $zic -d $output/right -L $leapseconds $i #2>> $build_dir/stderr
   cat $build_dir/stderr | grep -v "time zone abbreviation differs from POSIX standard" >&2 || true
```
Comment (Tomin1) [inline scripts/zone-generate.sh:88]: If you don't direct the output to the file, there is probably no point keeping this line.

Comment (pvuorela) [reply scripts/zone-generate.sh:88]: Actually that redirection removal i was only supposed to do for testing. I'll get it back.

File: rpm/tzdata-timed.spec [hunk 2]
```diff
@@ -1,12 +1,11 @@
 Name:       tzdata-timed
 
 Summary:    Data files for the time daemon, timed
-Version:    2017b
+Version:    2021a
 Release:    1
-Group:      System/System Control
 License:    Public Domain
```
Comment (Tomin1) [inline rpm/tzdata-timed.spec:6]: Should this have a license file anyway?

**Review by Tomin1 — APPROVED** [review]
LGTM with a comment

File: rpm/tzdata-timed.spec (hunk 2 above)
Comment (pvuorela) [reply rpm/tzdata-timed.spec:6]: Hm, technically there is no license if it's public domain? Some COPYING type of file dunno, but that's separate anyway from the update.

Comment (Tomin1) [reply rpm/tzdata-timed.spec:6]: It's just that to me not having a license of any kind sounds problematic.

**Pull request merged**